src/components/shared/Extensions/Tooltip/Tooltip.tsx

```diff
@@ -53,7 +54,16 @@ const Tooltip: FC<PropsWithChildren<TooltipProps>> = ({
               }`,
             })}
           />
-          <div className="max-w-[15.625rem] flex flex-col items-start">{tooltipContent}</div>
+          <div className="max-w-[15.625rem] flex flex-col items-start">
+            {isSuccess ? (
+              <div className="flex items-center [&>i>svg]:fill-base-white gap-2">
```

Comment:
`[&>i>svg]:fill-base-white` --> `text-base-white`
Please use 'color' properties to svg icons (not fill or stroke).